packages/card/AvatarCard.jsx

```diff
@@ -34,21 +34,21 @@ export default function AvatarCard({
     <article
       aria-label={title}
       data-test="avatar-card"
-      className="su-component-card su-relative su-w-full md:su-basis-1/3 su-flex su-flex-wrap su-gap-10 lg:su-content-start lg:su-max-w-[293px]"
+      className="su-component-card su-relative su-w-full md:su-basis-1/3 su-flex su-flex-wrap su-gap-10 lg:su-content-start lg:su-max-w-[29.3rem]"
     >
-      <h3 className="su-text-21 lg:su-text-24 su-leading-[25.2px] lg:su-leading-[28.8px] su-flex-grow su-my-0 su-font-serif su-w-full">
+      <h3 className="su-text-21 lg:su-text-24 su-leading-display su-flex-grow su-my-0 su-font-serif su-w-full">
         <XssSafeContent
-          className="focus:su-outline-0 focus:su-ring su-transition su-text-black dark:su-text-white dark:hover:su-text-dark-mode-red su-font-bold su-no-underline hover:su-text-digital-red dark:su-text-white dark:hover:su-text-dark-mode-red"
+          className="su-transition su-text-black su-font-bold su-no-underline hocus:su-text-digital-red dark:su-text-white dark:hocus:su-text-dark-mode-red hocus:su-underline"
           content={title}
           href={liveUrl}
           elementType="a"
         />
       </h3>
       {authorDisplayName && (
         <div
-          className={`su-flex su-w-full su-min-h-[56px] su-self-end lg:su-self-start su-items-center su-gap-10 ${
-            authorAvatar ? "su-ml-[-3px] su-mb-[-3px]" : ""
-          } su-text-black dark:su-text-white su-text-16 su-leading-[19.106px]`}
+          className={`su-flex su-w-full su-min-h-[5.6rem] su-self-end lg:su-self-start su-items-center su-gap-10 ${
+            authorAvatar ? "su--ml-3 su--mb-3" : ""
+          } su-text-black dark:su-text-white su-text-16 su-leading-display`}
```

Comment:
Since I'm already in this file, I did a little bit of clean up with the TW classes.